src/st/dre/1E1C8.c

```diff
@@ -29,8 +29,8 @@ void EntityUnkId13(Entity* entity) {
     }
 }
 
-// DECOMP_ME_WIP func_8019E3C8 https://decomp.me/scratch/lcx4I
-INCLUDE_ASM("st/dre/nonmatchings/1E1C8", func_8019E3C8);
+// DECOMP_ME_WIP EntityUnkId14Spawner https://decomp.me/scratch/lcx4I
```

Comment:
If this function is decompiled we can remove the WIP comment

Reply:
Thanks, good catch!